### PR TITLE
Nineteen Eighty-Four

### PR DESCRIPTION
**Pull request by ashameimaru**
(This time I hope that i put it in the right branch!) This is her review of Nineteen Eighty-Four by George Orwell. Note that I tiptoed and went around anything political and went straight to what she felt connected the most.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4791,6 +4791,42 @@ label monika_urgent:
...
+	m "It was written by George Orwell."
+	m 4a "It is popularly thought about with surveillance and Big Brother. And said and used frequently in the debate club I left."
+	m "That is not what I want to bring up, [player]."
+	m "There was something different I got out of this~"
+	m 1c "Like how language is manipulated here? They called this manipulation Newspeak."
+	m 1r "The government in this book, Ingsoc, was creating this language to control people's thoughts by not just creating words."
+	m 1h "But destroying them! They were reducing grammar and vocabulary to bare basics so you couldn't even think of anything else but them."
+	m "Even to be reduced to not express your own feelings and emotions with yourself!"
+	m "There was this person that worked for them named Syme who said this:"
+	m 1r "'It's a beautiful thing, the destruction of words. Of course the great wastage is in the verbs and adjectives,
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `"` at the end of this line.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4791,6 +4791,42 @@ label monika_urgent:
...
+# I did not use the 'Big Brother','surveillance' keywords, 
+# because either they may get used on other topics
+	
+init 5 python:
+    for key in ['1984','Nineteen Eighty-Four','Ingsoc','newspeak','George Orwell']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_1984')
+    monika_random_topics.append('monika_1984')
+
+label monika_1984
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: your indentation for all the dialogue in this label is wrong. Please make sure that your indentation settings in whatever editor you are using are set to `spaces` and `4`. (We want 4 space indentations).

**Comment by ThePotatoGuy** [comment]
Despite the errors, I'm glad you're adding a 1984-related topic. About time we had one.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4791,6 +4791,42 @@ label monika_urgent:
...
     return
 
+# For George Orwell's Nineteen Eighty-Four, Monika avoids political usage,
+# as it reminds her of the debate club, but more the interpretation of language
+# and thought, which she would feel better connected with.
+# I did not use the 'Big Brother','surveillance' keywords, 
+# because either they may get used on other topics
+	
+init 5 python:
+    for key in ['1984','Nineteen Eighty-Four','Ingsoc','newspeak','George Orwell']:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: keys need to be lowecase. Please change your key strings to lowercase.

File: Monika After Story/game/script-topics.rpy (hunk 2 above)
Comment (ashameimaru) [reply Monika After Story/game/script-topics.rpy]: I have hopefully done the proper corrections of capitalization and spaces.

**Comment by ashameimaru** [comment]
Appropriate corrections have been placed, hopefully~

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4791,6 +4791,42 @@ label monika_urgent:
...
+# I did not use the 'big brother','surveillance', etc. keywords, because they may 
+# get used on other topics
+	
+init 5 python:
+    for key in ['1984','nineteen eighty-four','ingsoc','newspeak','george orwell']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_1984')
+    monika_random_topics.append('monika_1984')
+
+label monika_1984
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: You need a `:` after the label like `label this_label:`

Comment (ashameimaru) [reply Monika After Story/game/script-topics.rpy]: Colon added~

**Comment by monikLover** [comment]
Edited and reviewed. Approved.

**Pull request merged**